docs/source/contributor-guide/index.md

```diff
@@ -50,11 +50,11 @@ A "major" PR means there is a substantial change in design or a change in the AP
 3. Non-controversial build-related changes (clippy, version upgrades etc.)
 4. Smaller non-controversial feature additions
 
-# Developer's guide
+## Getting Started
```

Comment:
I changed some of these heading levels so they didn't all appear in the main table of contents on https://arrow.apache.org/datafusion/